News-Android-App/src/main/res/layout/activity_login_dialog.xml

```diff
@@ -0,0 +1,156 @@
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/username_container"
+                android:layout_alignParentStart="true"
+                android:id="@+id/password_container"
+                app:counterOverflowTextColor="@android:color/white">
+
+                <EditText
+                    android:id="@+id/password"
+                    android:gravity="start"
+                    android:textAlignment="viewStart"
+                    android:hint="@string/pref_title_password"
+                    android:inputType="textPassword"
+                    android:maxLines="1"
+                    android:singleLine="true"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:textColor="@color/material_grey_100"
+                    android:textColorHint="@android:color/white"/>
+
+            </com.google.android.material.textfield.TextInputLayout>
+
+            <com.google.android.material.textfield.TextInputLayout
+                android:id="@+id/url_container"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/password_container"
+                android:layout_alignParentStart="true"
+                app:counterOverflowTextColor="@android:color/white">
+
+                <EditText
+                    android:id="@+id/edt_owncloudRootPath"
+                    android:gravity="start"
+                    android:textAlignment="viewStart"
+                    android:hint="@string/pref_title_owncloudRootPath"
+                    android:inputType="textUri"
+                    android:maxLines="1"
+                    android:selectAllOnFocus="true"
+                    android:singleLine="true"
+                    android:title="@string/pref_default_owncloudRootPath"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:textColor="@color/material_grey_100"
+                    android:textColorHint="@android:color/white"/>
+
+            </com.google.android.material.textfield.TextInputLayout>
+
+            <ImageView
+                android:id="@+id/imgView_ShowPassword"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="end"
+                android:paddingLeft="10dp"
+                android:paddingRight="10dp"
+                android:src="@drawable/ic_action_visibility"
+                android:layout_alignTop="@+id/password_container"
+                android:layout_alignBottom="@+id/password_container"
+                android:layout_alignEnd="@+id/password_container"
+                android:contentDescription="@string/content_desc_show_password"/>
+
+            <androidx.appcompat.widget.AppCompatCheckBox
```

Comment:
```suggestion
            <CheckBox
```

See https://developer.android.com/reference/android/support/v7/widget/AppCompatCheckBox
> This will automatically be used when you use CheckBox in your layouts and the top-level activity / dialog is provided by appcompat. You should only need to manually use this class when writing custom views.